### ISSUE TITLE
[İstanbul] Data Scientist

### ISSUE THREAD

**Issue by Dutegani** [CASTECH-Software-Solutions/open_roles#35]
> English: Advanced
> Employment status: Full-time and Remote 
> Location: İstanbul, İstanbul, Turkey

## About the company

AFINITI
Create happier, more loyal customers and agents
Each time a customer interacts with your brand is an opportunity to win their trust.
Get that customer talking to a customer service representative they click with, and you increase the chances of a positive outcome from the conversation.
The result? That customer will stay with you for longer, spend more, and broadcast the best of your brand to the world.
At Afiniti, we can help you make this happen. Our patented pairing technology puts your customers in touch with the agents uniquely best-suited to helping them, transforming the quality of interactions for all involved.

https://www.afiniti.com/what-we-do 

 ## About the project
Our client is the leading artificial intelligence solutions provider for enterprise retailers. They are passionate about helping retailers unlock valuable, transformative business insights from their data through their innovative scalable technology.

In the last 5 years, our client has experienced exponential growth having worked with many of today’s Fortune 50 retailers, who rely on them for their AI-enabled forecasting, promotion and assortment and allocation solutions.

This role is expected to participate in design and implementation of client projects, as well as drive the communication and interaction with customers. Primarily focused on projects that leverage Forecast as a Service (FaaS) technology, the expectation is that much of their time is focused on data engineering, feature set creation, training forecast models, forecast accuracy scorecards and other analytical activities to support FaaS.

### **Job Reqirements:** 
- You will help them to apply a wide range of statistical and quantitative analysis in practical and valuable ways.
- You will be processing, cleansing, and verifying the integrity of data used for analysis.
- You will be selecting features, implementing forecast solutions using their Retail Science Platform.
- You will participate in design workshops and document business requirements and constraints.
- Perform detailed data analysis to validate that it conforms to expectations.
- Conduct tuning analysis to support development and configuration decisions.
- Create and maintain rich interactive visualizations through data interpretation and analysis integrating various reporting components from multiple data sources.
- Program and write maintainable and well-documented scripts/applications for process automation.
- Learn and work with subject matter experts.

### **Candidate Requirements:**
- Experience with Python (Jupyter, NumPy, Pandas, PySpark) preferred, must have experience in at least one of common data science toolkits, such as scikit learn, Tensorflow, PyTorch, etc
- Strong SQL or Excel skills with the ability to learn other analytic tools;
- Fluent English skills in writing snd speaking.
- Degree in Comp. Science, Engineering, Math, Statistics, Research or a related field;
- Ability to take on responsibilities and work with minimal supervision;
- Ability to work on multiple projects;
- Ability to create and maintain outstanding partner relationships;
- Strong problem solving and analytical skills including mining, evaluation, analysis, and visualization;
- Excellent written and verbal communication skills including reports and presentations;
- A drive to learn and master new technologies and techniques.

## Benefits
- In addition to join them to build state-of-the-art, AI-enabled software we also get a ton of perks!
- You get the choice to work on a Mac or a PC.
- Casual dress code, social events and after-works. Flexible, diverse work environment. Respectful, innovative team.
- But it’s not all about the fun. You get a competitive salary and a progressive bonus while getting a startup experience at a company with an awesome culture. You get 21 days of PTO, major national holidays.
